project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 object Dependencies {
 
   val test = Seq(
-    "uk.gov.hmrc"                %% "webdriver-factory"       % "0.39.0" % Test,
+    "uk.gov.hmrc"                %% "webdriver-factory"       % "0.40.0" % Test,
```

Comment:
Please address all the warnings in the file. You will need to upgrade other libraries too